### PR TITLE
Enforce max_len in corpus pruning task (follow up #1616).

### PR DESCRIPTION
**Pull request by Dor1s**
Otherwise the large elements living in GCS (or added there manually) will remain.

### CODE REVIEW

**Comment by Dor1s** [comment]
Haven't found any tests for this, let me see if GCB fails, maybe there is something I missed. Otherwise will have to add a new test.

File: src/python/bot/tasks/corpus_pruning_task.py [hunk 1]
```diff
@@ -338,25 +341,35 @@ def __init__(self, build_directory, context):
 
   def get_libfuzzer_flags(self):
```
Comment (oliverchang) [inline src/python/bot/tasks/corpus_pruning_task.py:342]: we should find a way to merge this with https://github.com/google/clusterfuzz/blob/32ce4d06bf75dabff18f009b5eacd3ae88e7fc51/src/python/bot/fuzzers/libFuzzer/fuzzer.py#L31.

The detect_leaks bit is also missing there.

File: src/python/bot/tasks/corpus_pruning_task.py [hunk 2]
```diff
@@ -81,8 +81,11 @@
 RSS_LIMIT = 2560
 RSS_LIMIT_MB_FLAG = '-rss_limit_mb=%d'
 
-# Flag to disable leak checking.
-DISABLE_LEAK_CHECK_FLAG = '-detect_leaks=0'
+# Flag to enforce length limit for a single corpus element.
+MAX_LEN_FLAG = '-max_len=%d'
```
[the remark this answers is not in the document]
Comment (Dor1s) [reply src/python/bot/tasks/corpus_pruning_task.py:85]: yep, there even seems to be a todo for that https://github.com/google/clusterfuzz/blob/e0c6ad071951c403d17ef11a3020ca75b4bf9a09/src/python/bot/tasks/corpus_pruning_task.py#L52

File: src/python/bot/tasks/corpus_pruning_task.py (hunk 1 above)
Comment (Dor1s) [reply src/python/bot/tasks/corpus_pruning_task.py:342]: The libFuzzer's one uses all the options provided in the `.options` file, with hich value sanitization for `timeout` and `rss_limit_mb`. The pruning one only extracts certain known options, nothing else.

I'm not sure if we want to use all the options for pruning as well. I mean, it seems reasonable in general, but I recall we had some targets that set a particular value for the `-runs` option (in Chromium, due to some threading issue), I don't think we would want that to be used in the pruning, so I guess there might be some other corner cases.

OTOH it's probably fault of the fuzz target developer who added some unlucky flag to the `.options` file, so maybe we shouldn't care. Having `close_fd_mask` to be turned out during pruning would be definitely better than not having it, for an opposite example.

**Pull request merged**